area_detector_handlers/eiger.py

```diff
@@ -0,0 +1,158 @@
+"""
+This code was originally developed in https://github.com/NSLS-II/eiger-io
+and copied (the lazy way, without retaining git history) into this repo.
+It has been substantially changed from the original.
+"""
+from glob import glob
+import os
+from pathlib import Path
+
+import dask.array
+import h5py
+
+from . import HandlerBase
+
+
+class EigerHandler(HandlerBase):
+    EIGER_MD_LAYOUT = {
+        'y_pixel_size': 'entry/instrument/detector/y_pixel_size',
+        'x_pixel_size': 'entry/instrument/detector/x_pixel_size',
+        'detector_distance': 'entry/instrument/detector/detector_distance',
+        'incident_wavelength': 'entry/instrument/beam/incident_wavelength',
+        'frame_time': 'entry/instrument/detector/frame_time',
+        'beam_center_x': 'entry/instrument/detector/beam_center_x',
+        'beam_center_y': 'entry/instrument/detector/beam_center_y',
+        'count_time': 'entry/instrument/detector/count_time',
+        'pixel_mask': 'entry/instrument/detector/detectorSpecific/pixel_mask',
+    }
+    specs = {'AD_EIGER2', 'AD_EIGER'}
```

Comment:
Is `specs` used anywhere? I think one has to declare it via entrypoints anyway.

Reply:
Since there is no way to do the reverse lookup ("What specs is this handler for?") I think it is still worth keeping the `specs` attribute. It is not used by databroker as far as I know, but it might be useful for downstream code.

Reply:
Sounds good.